text/0000-ownership-transfer.md

```diff
@@ -0,0 +1,120 @@
+- Feature Name: ownership-transfer
+- Start Date: 2020-3-19
+- RFC PR: (leave this empty)
+- Bundler Issue: (leave this empty)
+
+# Summary
+
+gem owner -a/r command is used to add or remove owners by an email. We will add options for adding/removing gem owners via Web UI. Additionally, we will make owner add/remove events notifiable, i.e. users can opt-in to receive email notification about ownership changes of a gem. 
+
+We will also add a new flow for ownership transfers. Any gem owner `looking for maintainers` will be able to create `ownership request`. Any user can make an `ownership application` for any gem with less than 100,000 downloads and older than 12 months updated at time or gems which are marked as looking for maintainers. When a gem owner approves an `ownership application`, the requester would be added as an owner to the gem. These `ownership requests` will be searchable and listed on a site wide link.
```

Comment:
I'm not sure if I understand well the 100.000 downloads and 12 months updated limitations here.

Let's say I'm gem owner of popular but unmaintained gem breaking both those limits and I have not set gem in `looking for maintainers` mode. If I understand it well, nobody can make an `ownership application`. Is that any basic kind of "anti-spam" filter here?

Also I do not follow that creating `ownership request` is actually term for "looking for maintainers". `gem owner` creates `ownership request` sounds more like opposite case (asking to get ownership of 3rd party gem) IMHO. I'm not native :gb: speaker, but something like "call for maintainers" would be IMHO better terminology in here.

Last idea is about automated switch to `looking for maintainers` for gem matching some criteria or (less intrusive) to at least send friendly email to owners to consider marking gem as `looking for maintainers`.

Reply:
>  Is that any basic kind of "anti-spam" filter here?

yes, anti-spam and limiting impact of anyone replacing gem source with malicious code. If a significant number of users are downloading a gem, I would say it has enough credibility to stay on our site undisturbed (unless explicitly allowed by existing owners). 

> creating `ownership request` is actually term for "looking for maintainers"

my personal preference with using `ownership ..` in name is that it keep things aligned with existing ownership model.
I agree `ownership request` is a bit confusing, my PR on rg.org uses `adoption request` which is only slight better. However, the word `adoption` (or `looking for maintainers`) gives false the idea that existing gem will be maintained, when the new owner can replace the gem content if they chose to do so.
We are open to suggestion for a better a name, which keep it clear that user being added with this new flow has same guarantees as `gem owner -a/r`.

> send friendly email to owners to consider marking gem as looking for maintainers

a blog post will definitely be in order, not so sure about email to all users. It would depend on how good or bad the roll-out it.